app/Filament/Resources/GameResource.php

```diff
@@ -0,0 +1,340 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Filament\Resources;
+
+use App\Filament\Extensions\Resources\Resource;
+use App\Filament\Resources\GameResource\Pages;
+use App\Filament\Rules\ExistsInForumTopics;
+use App\Filament\Rules\IsAllowedGuideUrl;
+use App\Models\Game;
+use App\Models\System;
+use App\Models\User;
+use Filament\Forms;
+use Filament\Forms\Form;
+use Filament\Infolists;
+use Filament\Infolists\Infolist;
+use Filament\Pages\Page;
+use Filament\Tables;
+use Filament\Tables\Table;
+use Illuminate\Contracts\Support\Htmlable;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletingScope;
+use Illuminate\Support\Facades\Auth;
+
+class GameResource extends Resource
+{
+    protected static ?string $model = Game::class;
+
+    protected static ?string $navigationIcon = 'fas-gamepad';
+
+    protected static ?string $navigationGroup = 'Platform';
+
+    protected static ?int $navigationSort = 1;
+
+    protected static ?string $recordTitleAttribute = 'title';
+
+    protected static int $globalSearchResultsLimit = 5;
+
+    /**
+     * @param Game $record
+     */
+    public static function getGlobalSearchResultTitle(Model $record): string|Htmlable
+    {
+        return $record->title;
+    }
+
+    /**
+     * @param Game $record
+     */
+    public static function getGlobalSearchResultDetails(Model $record): array
+    {
+        return [
+            'ID' => $record->id,
+            'System' => $record->system->name,
+        ];
+    }
+
+    public static function getGloballySearchableAttributes(): array
+    {
+        return ['ID', 'Title'];
+    }
+
+    public static function infolist(Infolist $infolist): Infolist
+    {
+        return $infolist
+            ->schema([
+                Infolists\Components\ImageEntry::make('badge_url')
+                    ->label('')
+                    ->size(config('media.icon.lg.width')),
+
+                Infolists\Components\Section::make('Metadata')
+                    ->columns(['md' => 2, 'xl' => 3, '2xl' => 4])
```

Comment:
Not sure if there's an easy way to do it, but published achievements is arguably more important that ForumTopicId for the fourth column.

Reply:
Very easy - done in latest.

Reply:
Looks like the column disappeared entirely?
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/fe1847e0-112a-4ae2-86cb-1406f189678c)

Reply:
Hrm, this is what I see:

<img width="1441" alt="Screenshot 2024-06-07 at 8 28 27 PM" src="https://github.com/RetroAchievements/RAWeb/assets/3984985/c3f301bb-d932-4a0c-ab76-c577ec2b59ab">

Does anything change if you do `sail artisan view:clear`?

Reply:
Seems to have been a cookie or something. I logged in incognito mode and it was present, so I logged out and logged back in normal mode and it's present there too now.